frontend/base/src/main/resources/GitMacheteBundle.properties

```
@@ -117,18 +116,22 @@ action.GitMachete.BaseResetBranchToRemoteAction.notification.title=Resetting
 action.GitMachete.BaseResetBranchToRemoteAction.notification.title.reset-fail=Reset of branch ''{0}'' failed
 action.GitMachete.BaseResetBranchToRemoteAction.notification.title.reset-success.HTML=<html>Branch <b>{0}</b> has been reset to remote</html>
 action.GitMachete.BaseResetBranchToRemoteAction.task-title=Resetting branch
-action.GitMachete.BaseResetBranchToRemoteAction.task-subtitle=Resetting...
+action.GitMachete.BaseResetBranchToRemoteAction.task-subtitle=Resetting\u2026
 
 action.GitMachete.BaseSlideInBranchBelowAction.description=Slide in branch below ''{0}''
 action.GitMachete.BaseSlideInBranchBelowAction.description.disabled.no-parent=Slide in disabled due to undefined parent branch
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.create-new-branch.title=Create New Branch From {0}
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.checkbox.reattach=Reattach children
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.branch-name=Branch name:
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.parent=Parent Branch:
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.parent=Parent branch:
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.ok-button=Slide In
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.title=Slide In Branch Below
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.error.slide-in-under-itself=Cannot slide in the branch under itself
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.error.slide-in-under-its-descendant=Cannot slide in the branch under any of its descendants
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.no-branch-selected=The branch should be selected.
```

Or rather
```suggestion
action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.no-branch-selected=A branch should be selected.
```
❓ 

As in, _any_ branch, not _a particular_ branch